### ISSUE TITLE
Add support for pg-native

### ISSUE THREAD

**Issue by andrei-cocorean** [vitaly-t/pg-promise#36]
Hi, did you consider adding support for [pg-native](https://github.com/brianc/node-pg-native)? Since it's meant to be a drop-in replacement for pg, a flag in the init phase should do the trick.

**Comment by vitaly-t** [comment]
I haven't, but what would be the advatage of supporting it, you think?

**Comment by andrei-cocorean** [comment]
In some cases you might want the boost in performance from the native bindings and it's easier to make the switch if pg-promise supports it.

**Comment by vitaly-t** [comment]
Speaking of boost in performance, I've just released [v.1.8.0](https://github.com/vitaly-t/pg-promise/releases/tag/v.1.8.0) to support [pg-query-stream](https://github.com/brianc/node-pg-query-stream), which by the way doesn't even work with pg-native, which makes it of less value, I suppose.

I will look at testing against it though, to see if it even works...

**Comment by andrei-cocorean** [comment]
Thanks for looking into this. I'll give it a try when I have some time.

**Comment by vitaly-t** [comment]
@andrei-cocorean , ok, if you do, I will re-open the issue, but closing it for the time being.

**Comment by vitaly-t** [comment]
@andrei-cocorean did you have a chance to look at it? I did, and found nothing but trouble, trying to use the native bindings, especially under Windows. Many of those were reported to the author of node-postgres, but every time he came back with the same lame excuse that he doesn't have a Windows machine to fix it. At some point I just stopped trying to use the native bindings, it doesn't work.

As a matter of fact, I've been thinking it over more than once - rewriting the entire low-level driver (node-postgres), because I'm not satisfied with how the existing one is supported, but I'm yet to find the time to embark on something this massive. I only know that it works in 99%, without the native bindings. The remaining 1% refers to all the known issues that have been ignored for a long time.

To make the matters worse, the author of node-postgres recently black-listed my account, his biggest supporter, so I could no longer provide any more support for node-postgres. What an a-hole, truly!
